src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -2,12 +2,12 @@
   "persons": [ {
     "name": "Valid Person",
     "phone": "9482424",
-    "email": "hans@example.com",
+    "cycle": "Summer 2024",
```

Comment:
Nice! Perhaps sample data like this one could be implemented in the other parts that you've changed as well!